syntax = "proto3";

option cc_generic_services = true;

import "google/protobuf/empty.proto";
import "service/DataService.proto";
package apb.api.characterviewer;

option java_package = "jp.co.apb.server.api.proto.characterViewer";
option java_multiple_files = true;
option csharp_namespace = "Art.Framework.ApiNetwork.Grpc.Api.CharacterViewer";

message CharacterViewerTopResponse {
  repeated int32 release_character_viewer_field_id = 1;
  map<string, .apb.api.data.DiffData> diff_user_data = 99;
}

service CharacterViewerService {
  rpc CharacterViewerTop(.google.protobuf.Empty) returns (.apb.api.characterviewer.CharacterViewerTopResponse);
}

